test/parser/parser_test.exs

```diff
@@ -28,15 +28,20 @@ defmodule Membrane.H264.FFmpeg.Parser.Test do
     assert {:ok, state} = Parser.handle_caps(:input, input_caps, nil, state)
     assert state.frame_prefix == expected_prefix
 
-    # Check with stream that doesn't have pps and sps
-    assert {{:ok, actions}, %{frame_prefix: nil}} =
-             Parser.handle_process(:input, %Buffer{payload: @no_params_stream}, nil, state)
+    # Check with stream that doesn't have pps and sps - first buffer too short
```

Comment:
How about adding the check with first buffer too short as a separate test?

Reply:
Yeah, good point. Actually, I think there is more than one test to be extracted from here